### PR TITLE
Sink and Sink Lightwieght  Testflows Integration

### PR DESCRIPTION
**Pull request by AlmostIvanSidorov**
PR to check sink and sink lightwieght testflows integration tests and new altinity runners.

### CODE REVIEW

File: .github/workflows/testflow_tests_lightwiegth.yml
Comment (subkanthi) [inline .github/workflows/testflow_tests_lightwiegth.yml:1]: can u please fix the typo in the filename

File: sink-connector-lightweight/docker/docker-compose.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ services:
       - "8123:8123"
       - "9000:9000"
     environment:
-      - CLICKHOUSE_USER=ch_user
```
Comment (subkanthi) [inline sink-connector-lightweight/docker/docker-compose.yml]: can u please rollback these changes.

File: sink-connector-lightweight/docker/docker.env [hunk 2]
```diff
@@ -2,10 +2,10 @@ export database.hostname="mysql-master"
 export database.port="3306"
 export database.user="root"
 export database.password="root"
-export database.include.list=sbtest
+export database.include.list=test
```
Comment (subkanthi) [inline sink-connector-lightweight/docker/docker.env]: Also please rollback this.

**Review by subkanthi — CHANGES_REQUESTED** [review]

File: sink-connector-lightweight/docker/docker.env (hunk 2 above)
Comment (AlmostIvanSidorov) [reply sink-connector-lightweight/docker/docker.env]: Yes, at the moment I am testing runners, I will remove all discrepancies with the main project, except for my tests when everything is ready.

**Pull request merged**